HicServices/RDMP
# Task/rdmp 155 migrate catalogue tables

Allows users to tell REDMP that the data a catalogue uses is in a different database/table

How to test:
* create a secondary table (back up your example data and create a new table from it)
* Use the functionality (see docs) to migrate catalogue to new location
* Check that columnInfo and extractionInfo are correctly updated
